pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -214,3 +262,32 @@ func (ct *connTrackOvsCtl) GetMaxConnections() (int, error) {
 	}
 	return maxConns, nil
 }
+
+func statusStringToStateflag(status string) uint32 {
+	// Mapping is defined at https://github.com/torvalds/linux/blob/master/include/uapi/linux/netfilter/nf_conntrack_common.h#L42
```

Comment:
thanks, you should use a permanent link here: https://github.com/torvalds/linux/blob/v5.9/include/uapi/linux/netfilter/nf_conntrack_common.h#L42

Reply:
Thanks, fixed.